code/game/objects/items.dm

```diff
@@ -132,6 +132,21 @@ GLOBAL_DATUM_INIT(welding_sparks_prepdoor, /mutable_appearance, mutable_appearan
 
 	var/active = FALSE
 
+
+
+	//Coloring vars
+	///Some defines to determine if the item is allowed to be recolored.
+	var/colorable_allowed = NONE
+	///optional assoc list of colors we can color this item
+	var/list/colorable_colors = list()
+	///List of icon_state suffixes for item variants.
+	var/list/icon_state_variants = list()
```

Comment:
Please string list this or something

Reply:
pending

Reply:
It is string listed. Since string_list is a proc, it has to be set in Initialize, which is what I did.